src/LibExpat.jl

```diff
@@ -298,7 +298,7 @@ function find{T<:AbstractString}(pd::ETree, path::T)
         end
     end
 
-    xp= Array(Tuple{Symbol,Any},0)
+    xp= Array{Tuple{Symbol,Any}}(0)
```

Comment:
The other methods could also be instead written as `SymbolAny[]` and `returntype[]` Is that easier to read that the explicit array constructor? 

(fyi can you turn on branch protection for master, to be safe? the move  in the org and my fork being based on the amit's repo kinda screwed things up)